packages/walletkit/src/api/models/core/Primitives.ts

```diff
@@ -0,0 +1,41 @@
+
+export type Address = string; // User-friendly address representation
```

Comment:
Address is used in the @ton/core, lets name it FriendlyAddress or something like that